.ci/integration/tasks/dependencies.yml

```diff
@@ -28,3 +28,9 @@
     name: fpm
     state: present
     user_install: no
+
+# here we ignore a none zero status code as the package is installed
+- name: Install apk packaging dependencies for Alpine based distros
+  command: bash -lc "gem install etc || :"
```

Comment:
why the `|| :` ? Are we expecting `gem install etc` to fail sometimes?

Reply:
We are expecting it to succeed in installing `etc` then return a non-zero status code. That confused me for a while until I logged onto the Alpine docker image and confirmed that is what it does. It says it installed `etc` then barfs a complaint to stderr about some unrelated. Since it said it installed the missing package and the code that uses dosen't blow up with a missing package I chose to not to attempt attempt fix the gem system. The fix/build/test cycle is very slow to debug and wipe images to restart. This means that cleaning up ”fugly” things like this which annoy me feels excessively costly.

Reply:
i have changed this to read `# here the package is installed but a non-zero status code is returned that we ignore`